core/src/main/res/values/strings.xml

```diff
@@ -624,118 +624,118 @@
     <string name="statistics_space_used">Space used:</string>
-    <string name="statistics_view_all">View for all podcasts »</string>
+    <string name="statistics_view_all">For all podcasts »</string>
 
     <!-- AntennaPodSP -->
-    <string name="sp_apps_importing_feeds_msg">Importing subscriptions from single-purpose apps&#8230;</string>
+    <string name="sp_apps_importing_feeds_msg">Importing subscriptions from single-purpose apps…</string>
 
     <!-- Add podcast fragment -->
-    <string name="search_podcast_hint">Search podcast…</string>
-    <string name="search_itunes_label">Search iTunes</string>
-    <string name="search_podcastindex_label">Search Podcastindex.org</string>
-    <string name="search_fyyd_label">Search fyyd</string>
+    <string name="search_podcast_hint">Search for podcast…</string>
+    <string name="search_itunes_label">Search on iTunes</string>
+    <string name="search_podcastindex_label">Search on Podcastindex.org</string>
+    <string name="search_fyyd_label">Search on fyyd</string>
     <string name="advanced">Advanced</string>
-    <string name="add_podcast_by_url">Add Podcast by RSS address</string>
-    <string name="add_podcast_by_url_hint" translatable="false">www.example.com/feed</string>
+    <string name="add_podcast_by_url">Add podcast RSS address</string>
+    <string name="add_podcast_by_url_hint">www.example.com/feed</string>
     <string name="browse_gpoddernet_label">Browse gpodder.net</string>
     <string name="discover">Discover</string>
     <string name="discover_hide">Hide</string>
-    <string name="discover_is_hidden">You selected to hide suggestions.</string>
+    <string name="discover_is_hidden">You hid suggestions.</string>
     <string name="discover_more">more »</string>
-    <string name="discover_powered_by_itunes">Suggestions by iTunes</string>
+    <string name="discover_powered_by_itunes">Suggestions from iTunes</string>
     <string name="search_powered_by">Results by %1$s</string>
 
     <!-- Local feeds -->
     <string name="add_local_folder">Add local folder</string>
     <string name="local_folder">Local folder</string>
     <string name="reconnect_local_folder">Re-connect local folder</string>
-    <string name="reconnect_local_folder_warning">In case of permission denials, you can use this to re-connect to the exact same folder. Do not select another folder.</string>
-    <string name="local_feed_description">This virtual podcast was created by adding a folder to AntennaPod.</string>
-    <string name="unable_to_start_system_file_manager">Unable to start system file manager</string>
+    <string name="reconnect_local_folder_warning">Use this to re-connect to the same folder. Do not select another folder if you want to refresh permissions.</string>
+    <string name="local_feed_description">A virtual podcast created by adding a folder to AntennaPod.</string>
+    <string name="unable_to_start_system_file_manager">Could not start the system file-manager</string>
```

Comment:
Why these dashes? I have never seen them in english texts

Reply:
Because it isn't the manager of the system file.